Where_Is_My_Wife/Assets/_Main/Scripts/Zenject/MainInstaller.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEngine;
+using WhereIsMyWife.Controllers;
+using WhereIsMyWife.Managers;
+using WhereIsMyWife.Managers.Properties;
+using WhereIsMyWife.Player.State;
+using WhereIsMyWife.Player.StateMachine;
+using Zenject;
+
+namespace WhereIsMyWife
+{
+    public class MainInstaller : MonoInstaller
+    {
+        [SerializeField] private PlayerProperties _playerProperties;
+        
+        public override void InstallBindings()
+        {
+            InstallScriptableObjectBindings();
+            InstallManagerBindings();
+            InstallActionMethods();
+            InstallPlayerStates();
+        }
+
+        private void InstallManagerBindings()
+        {
+            Container.BindInterfacesTo<PlayerManager>().AsSingle();
+            Container.BindInterfacesTo<InputEventManager>().AsSingle();
```

Comment:
These are going to be `MonoBehavior`

Reply:
a9656178b45653044e6dfca1685da3128d18dad1